python/plugins/processing/modeler/ModelerParametersDialog.py

```diff
@@ -231,8 +231,10 @@ def showAdvancedParametersClicked(self):
             self.advancedButton.setText(self.tr('Show advanced parameters'))
         for param in self._alg.parameterDefinitions():
             if param.flags() & QgsProcessingParameterDefinition.FlagAdvanced:
-                self.wrappers[param.name()].widget.setVisible(self.showAdvanced)
-                self.wrappers[param.name()].label.setVisible(self.showAdvanced)
+                wrapper = self.wrappers[param.name()]
+                if issubclass(wrapper.__class__, QgsProcessingModelerParameterWidget):
```

Comment:
Shouldn't this be `if not issubclass....`? QgsProcessingModelerParameterWidget doesn't have .widget/.label

Reply:
Thanks! Let me know if I've missed an API break and I'll fix.

Reply:
no... happen for boolean widget wrapper for grass algorithms (in my case r.null)

Reply:
hmmmm but BooleanWidgetWrapper creates the widget...

Reply:
replicable creating a model with r.null grass algorithm.